js/src/product-feed/review-request/index.js

```diff
@@ -29,18 +30,36 @@ const ReviewRequest = ( { account = {} } ) => {
 		return null;
 	}
 
+	const handleNoticeClick = () => {
+		setModalActive( true );
+		recordEvent( 'gla_modal_open', { context: 'request_review' } );
+	};
+
+	const handleModalClose = ( action ) => {
+		setModalActive( false );
+		recordEvent( 'gla_modal_closed', {
+			context: 'request_review',
+			action,
+		} );
+	};
+
+	const handleReviewRequest = () => {
+		handleModalClose( 'request_review' );
```

Comment:
💅 What about using another `action` string to make it a little differentiated from `context`? Such as `confirm` or `confirm_request_review`.

![image](https://user-images.githubusercontent.com/17420811/156698994-373a74b6-793d-40c3-b68b-072bffec033c.png)